docs/source/data-access-api/index.rst

```diff
@@ -16,21 +16,195 @@ that the full implementation of the functionality is yet to be completed.*
 
 The API can be accessed from the following URL [WIP]: https://api.dev.imap-mission.com
 
+Command Line Utility
+====================
+To Install
+----------
+
+Run the following command to use the API CLI.
+
+    .. code-block:: bash
+
+        pip install imap-data-access
+
+Base Command Arguments
+----------------------
+
+The following are base command arguments for the CLI:
+
+    .. code-block:: bash
+
+        imap-data-access -h # or
+        imap-data-access query # or
+        imap-data-access download # or
+        imap-data-access upload
+
+Add the -h flag with any base command for more information on use and functionality.
+
+.. openapi:: openapi.yml
+   :group:
+   :include: /query
+
+**Example Usage:**
+
+.. code-block:: bash
+
+    imap-data-access query --start-date 20240101 --end-date 20241231 --output-format json
+    # The following line is returned:
+    [{'file_path': 'imap/swe/l0/2024/01/imap_swe_l0_sci_20240105_v001.pkts', 'instrument': 'swe', 'data_level': 'l0', 'descriptor': 'sci', 'start_date': '20240105', 'version': 'v001', 'extension': 'pkts'}, {'file_path': 'imap/swe/l0/2024/01/imap_swe_l0_sci_20240105_v001.pkts', 'instrument': 'swe', 'data_level': 'l0', 'descriptor': 'sci', 'start_date': '20240105', 'version': 'v001', 'extension': 'pkts'}]
+
+
+.. openapi:: openapi.yml
+   :group:
+   :include: /download
```

Comment:
Similar to the last comment, this is describing the REST API so should also be moved (same with the `/upload` section.